ckan/logic/action/update.py

```diff
@@ -306,11 +306,11 @@ def package_update(
         model.Session.rollback()
         raise ValidationError(errors)
 
-    if not resources_only:
-        #avoid revisioning by updating directly
-        model.Session.query(model.Package).filter_by(id=pkg.id).update(
-            {"metadata_modified": datetime.datetime.utcnow()})
-        model.Session.refresh(pkg)
+    #if not resources_only:
```

Comment:
Why is this being removed? It is useful for reducing lock contention when reordering resources.